packages/angular-material/test/autocomplete-control.spec.ts

```diff
@@ -99,104 +101,95 @@ describe('Autocomplete control Base Tests', () => {
   });
 
   it('should render', fakeAsync(() => {
-    const mockSubStore = setupMockStore(fixture, { uischema, schema, data });
-    mockSubStore.complete();
-    fixture.detectChanges();
+    setupMockStore(fixture, { uischema, schema, data });
+    getJsonFormsService(component).updateCore(
```

Comment:
Why do we need to do this? Doesn't `setupMockStore` not already setup the `JsonFormsService`?